scripts/helper_numpy.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# coding: utf8
+'''Compute a Numpy friendly data-structures from VGGish files
+Example
+-------
+$ cd {repo_root}
+$ ./scripts/helper_numpy.py --csv_file /path/to/sparse-labels.csv
```

Comment:
would be good to add a ` \` at the end of this line in case anyone tries to copy-paste this into bash

Reply:
Sure. One question about that line though. I followed the template you used in https://github.com/cosmir/openmic-2018/blob/master/scripts/featurefy.py#L11, but why are you not adding `python` at the beginning of the line? I.e:
`$python  ./scripts/featurefy.py --file /some/audio/file.wav ./output_dir`
I see that you do this example script often, but is that supposed to work without the python command at the beginning?

Reply:
there are two things you have to do to call the script as an executable (which is happening here):
* add the [shebang](https://stackoverflow.com/questions/2429511/why-do-people-write-the-usr-bin-env-python-shebang-on-the-first-line-of-a-pyt) line at the top of the file, specifying the appropriate interpreter
* run `chmod +x <filename>` to tell the filesystem the script is executable

if you run `ls -lha .` on the directory, you'll see that the executable bits are set for all users

Reply:
Alright. And what is the reason to do so and not just use the `python ...` command line? This seems more intuitive _as an example script_ as it can work right away for the user. Asking around me, I can tell a few people will be confused with calling the script as an executable.